Cargo.toml

```diff
@@ -27,7 +27,7 @@ nom = "5"
 mysql_common = "0.22"
 byteorder = "1"
 chrono = "0.4"
-time = "=0.2.7"
+time = "0.2.25"
```

Comment:
Ah, I see what happened — they backported the fix from `time 0.3` to `time 0.2.23` (https://github.com/time-rs/time/issues/292) :tada: I think listing `0.2.23` here _should_ be sufficient. But `0.2.25` is fine.